stdpopsim/genomes.py

```diff
@@ -309,89 +316,124 @@ def species_contig(
     def length(self):
         return self.recombination_map.sequence_length
 
-    @property
-    def all_intervals_array(self):
+    def dfe_breakpoints(self):
```

Comment:
I've renamed this, and made it return something different: a tuple rather than a dictionary, and a vector of breakpoints instead of an array of intervals. The reason for returning a tuple is that it's more pythonic (I think?). The reason for returning a vector of breakpoints is that (a) this is *easier* to use in the only place in the code we use this method, and (b) this is more useful because it's just what tskit statistics wants.